argus/backend/service/views.py

```diff
@@ -177,6 +178,19 @@ def resolve_groups_for_tests(self, tests: list[ArgusTest]):
 
         return releases
 
+    def get_pytest_view_results(self, view_id: str | UUID) -> list[PytestResultTable]:
+
+        view: ArgusUserView = ArgusUserView.get(id=view_id)
+        tests: list[ArgusTest] = []
+        for batch in chunk(view.tests):
+            tests.extend(ArgusTest.filter(id__in=batch).all())
+        tests = [test for test in tests if test.plugin_name == "generic"]
+        results = []
+        for batch in chunk(tests):
+            results.extend(PytestResultTable.filter(test_id__in=[t.id for t in batch]).allow_filtering().all())
```

Comment:
I'm not sure if I understand it correctly:
having scylla-master tests, after a month this will return 100s thousands results?